content/working_with/console/widgets/blueprintCatalog.md

```diff
@@ -25,6 +25,7 @@ These parameters are a must if you want to configure the widget to access privat
 ## Settings
 
 * `Blueprint Examples URL` - Specifies the JSON file from which the blueprints are being read
+* `List of fields to show` - The fields that are presented in the blueprints list. Default: Creator, State, Deployments.
```

Comment:
Have you checked the code?

It seems to me that the default is: Name, Description, Created, Updated.

Reply:
I checked right now, you are right:
<img width="557" alt="Screen Shot 2022-03-23 at 10 09 03" src="https://user-images.githubusercontent.com/17104212/159652555-a20a4321-023f-4d9c-b990-368323038e33.png">

Reply:
I updated it: [27d7c93](https://github.com/cloudify-cosmo/docs.getcloudify.org/pull/2137/commits/27d7c93de07b9e2983ede97fff74af61423ecc42)